### PR TITLE
[#121960325] Enforce Ruby 1.9 Hash Syntax

### PR DESCRIPTION
**Pull request by ryanere**
**Description:**
- Enforce Ruby 1.9 Hash Syntax
- Related [Rtopia PR](https://github.com/liftopia/rtopia/pull/1700)

**Acceptance criteria:**

``` qa
Styleguide makes sense.
```

**PT stories:** 
- https://www.pivotaltracker.com/story/show/121960325

RFC @liftopia/engineering

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -910,21 +910,21 @@ in inheritance.
...
   used.
 
     ```Ruby
     # bad
-    hash = { one: 1, two: 2, three: 3 }
-    
+    hash = { :one => 1, :two => 2, :three => 3 }
+
     # good
     hash = { :one => 1, :two => 2, :three => 3 }
```
Comment (jcarouth) [inline README.md]: WAT?

File: README.md [hunk 2]
```diff
@@ -958,15 +958,15 @@ in inheritance.
       :delta => 4,
       :alpha => 1,
```
Comment (jcarouth) [inline README.md]: You need to update all of the other examples (like this one) that use hashes to use the 1.9 syntax.

File: README.md (hunk 1 above)
Comment (ryanere) [inline README.md]: Derp.

File: README.md (hunk 2 above)
Comment (ryanere) [inline README.md]: Is there rubocop for READMEs? 🚎

**Comment by ryanere** [comment]
@jcarouth Did I miss anything?

**Comment by jcarouth** [comment]
@ryanere Looks like you got all of those. Although this doesn't make any sense:

> Multi-line hashes should be aligned on the hashbang (=>).

Maybe should reword to "Multi-line hashes should be aligned on the value."

**Comment by ryanere** [comment]
Thanks for proofreading @jcarouth.  All good now?

**Comment by jcarouth** [comment]
Sure. Amazing. 🍇

**Pull request merged**